CHANGELOG.md

```diff
@@ -2,20 +2,56 @@
 
 This file documents all notable changes to Juttle. The release numbering uses [semantic versioning](http://semver.org).
 
+## 0.6.0
+
+Released 2016-03-09
+
+### Major Changes
+
+- Enabled runaway program detection. [[#473](https://github.com/juttle/juttle/issues/473)]
+
+### Minor Changes
+
+- The `.` operator can now be used to access object properties (not just module exports). [[#497](https://github.com/juttle/juttle/pull/497)]
```

Comment:
I think it'd be useful to clarify what situations this extends the language to support nested points support for, i.e. rvalues for `put`, assignments in functions, etc.

Depending on how useful this is, I'd consider this a major change perhaps.